### PR TITLE
Release 4.3.0

### PR DESCRIPTION
**Pull request by davimacedo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,50 @@
...
-- NEW (EXPERIMENTAL): Idempotency enforcement for client requests. This deduplicates requests where the client intends to send one request to Parse Server but due to network issues the server receives the request multiple times. **Caution, this is an experimental feature that may not be appropriate for production.** [#6744](https://github.com/parse-community/parse-server/issues/6744). Thanks to [Manuel Trezza](https://github.com/mtrezza).
+[Full Changelog](https://github.com/parse-community/parse-server/compare/4.3.0...master)
+
+### 4.3.0
+[Full Changelog](https://github.com/parse-community/parse-server/compare/4.2.0...4.3.0)
+- PERFORMANCE: Optimizing pointer CLP query decoration done by DatabaseController#addPointerPermissions [#6747](https://github.com/parse-community/parse-server/pull/6747). Thanks to [mess-lelouch](https://github.com/mess-lelouch).
+- SECURITY: Fix security breach on GraphQL viewer. Thanks to [Antoine Cormouls](https://github.com/Moumouls).
+- IMPROVE: Remove wontfix label from stalebot [#6810](https://github.com/parse-community/parse-server/pull/6810). Thanks to [Diamond Lewis](https://github.com/dplewis).
```
Comment (TomWFox) [inline CHANGELOG.md:10]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,50 @@
...
+- IMPROVE: Update bug report template [#6805](https://github.com/parse-community/parse-server/pull/6805). Thanks to [Tom Fox](https://github.com/TomWFox).
```
Comment (TomWFox) [inline CHANGELOG.md:11]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,50 @@
...
+- NEW: Allow passing custom gql schema function to ParseServer#start options [#6762](https://github.com/parse-community/parse-server/pull/6762). Thanks to [Luca](https://github.com/lucatk).
+- NEW: Allow custom cors origin header [#6772](https://github.com/parse-community/parse-server/pull/6772). Thanks to [Kevin Yao](https://github.com/kzmeyao).
+- FIX: Fix context for cascade-saving and saving existing object [#6735](https://github.com/parse-community/parse-server/pull/6735). Thanks to [Manuel](https://github.com/mtrezza).
+- NEW: Add file bucket encryption using fileKey [#6765](https://github.com/parse-community/parse-server/pull/6765). Thanks to [Corey Baker](https://github.com/cbaker6).
+- FIX: Removed gaze from dev dependencies and removed not working dev script [#6745](https://github.com/parse-community/parse-server/pull/6745). Thanks to [Vincent Semrau](https://github.com/vince1995).
+- FIX: Fix Sponsors badge on README [#6711](https://github.com/parse-community/parse-server/pull/6711). Thanks to [Olle Jonsson](https://github.com/olleolleolle).
```
Comment (TomWFox) [inline CHANGELOG.md:28]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,9 +1,50 @@
...
+- FIX: Remove wrong console log in test [#6627](https://github.com/parse-community/parse-server/pull/6627). Thanks to [Gordon Sun](https://github.com/sunshineo).
+- IMPROVE: Graphql tools v5 [#6611](https://github.com/parse-community/parse-server/pull/6611). Thanks to [Yaacov Rydzinski](https://github.com/yaacovCR).
+- FIX: Catch JSON.parse and return 403 properly [#6589](https://github.com/parse-community/parse-server/pull/6589). Thanks to [Gordon Sun](https://github.com/sunshineo).
+- PERFORMANCE: Allow covering relation queries with minimal index [#6581](https://github.com/parse-community/parse-server/pull/6581). Thanks to [Noah Silas](https://github.com/noahsilas).
+- FIX: Fix Postgres group aggregation [#6522](https://github.com/parse-community/parse-server/pull/6522). Thanks to [Siddharth Ramesh](https://github.com/srameshr).
+- NEW: Remove Beta Tag from GraphQL API [#6565](https://github.com/parse-community/parse-server/pull/6565). Thanks to [Antoine Cormouls](https://github.com/Moumouls).
```
Comment (TomWFox) [inline CHANGELOG.md:46]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,9 +1,46 @@
...
+- FIX: Add production Google Auth Adapter instead of using the development url [#6734](https://github.com/parse-community/parse-server/pull/6734). Thanks to [SebC.](https://github.com/SebC99).
+- IMPROVE: Run Prettier JS Again Without requiring () on arrow functions [#6796](https://github.com/parse-community/parse-server/pull/6796). Thanks to [Diamond Lewis](https://github.com/dplewis).
+- IMPROVE: Run Prettier JS [#6795](https://github.com/parse-community/parse-server/pull/6795). Thanks to [Diamond Lewis](https://github.com/dplewis).
+- IMPROVE: Replace bcrypt with @node-rs/bcrypt [#6794](https://github.com/parse-community/parse-server/pull/6794). Thanks to [LongYinan](https://github.com/Brooooooklyn).
+- IMPROVE: Make clear description of anonymous user [#6655](https://github.com/parse-community/parse-server/pull/6655). Thanks to [Jerome De Leon](https://github.com/JeromeDeLeon).
+- IMPROVE: Simplify GraphQL merge system to avoid js ref bugs [#6791](https://github.com/parse-community/parse-server/pull/6791). Thanks to [Manuel](https://github.com/mtrezza).
```
Comment (Moumouls) [inline CHANGELOG.md:19]: May be a little error here :)

**Comment by Moumouls** [comment]
So many improvements & additions, here just a metrics that i found on NPM Stats, last year we got `72 000/month downloads` in June 2019 and this year we have reached the `100 000 downloads/month` in June 2020.

It seems that we are on the right way :)

File: CHANGELOG.md (hunk 5 above)
Comment (mtrezza) [reply CHANGELOG.md:19]: True, I didn't work on #6791, if you mean that?

Comment (davimacedo) [reply CHANGELOG.md:19]: Sorry guys... I missed out the message before merging. I will fix on the release and send a PR to fix on changelog.

**Pull request merged**